### ISSUE TITLE
When two fields have the same copy, content from only one is extracted

### ISSUE THREAD

**Issue by TheoChevalier** [wagtail/wagtail-localize#253]
Just noticed that issue with the prod homepage of https://give.thunderbird.net, because staging had no `seo_title` set.

The `Page title` (or `seo_title`) field in the Promote tab has the following copy: `Give To Thunderbird`

The `Title` field in the Content tab (so, the actual page title) has the same copy, `Give To Thunderbird`

Only the seo_title got extracted in the .pot file: https://github.com/mozilla-l10n/thunderbird-donate-content/blob/master/templates/pages/mozilla-donate.pot#L8

We should have had something like this:
```
msgctxt "seo_title"
msgid "Give to Thunderbird"
msgstr ""

msgctxt "title"
msgid "Give to Thunderbird"
msgstr ""
```

**Comment by kaedroho** [comment]
Fixed by #254

**Comment by TheoChevalier** [comment]
@kaedroho Should this be reopened until the PR is merged?
